### DISCUSSION TITLE
k8s deployment: Is there a way to add custom initContainers to user code location or run?

### DISCUSSION THREAD

**Discussion by dduong1603** [dagster-io/dagster#18734] [Q&A]
As title. Right now I don't think the user code location pod and the run pods have any initContainers. I would like to add some but I don't know how, or if it's even possible.

**Comment by dduong1603** [comment]
looks like for run pod you can use `podSpecConfig` if you use the `k8sRunLauncher`, but not sure if anything can be done for the user code deployments

**Comment by andrescolodrero** [comment]
if you want initContainers, the feature has been added recently:
https://github.com/dagster-io/dagster/pull/19192
